libs/sdk-ui-charts/src/charts/headline/internal/HeadlinePagination.tsx

```diff
@@ -0,0 +1,28 @@
+// (C) 2021 GoodData Corporation
+import React, { useState } from "react";
+
+import { HeadlinePaginationRenderer } from "./HeadlingePaginationRenderer";
+
+interface IHeadlinePaginationProps {
+    renderSecondaryItem?: () => JSX.Element;
```

Comment:
These don't seem to be optional, the component will blow up if these are not provided. Please make these mandatory.